src/main/java/ports/usecases/account/RemoveSemesterCourse/RemoveSemesterCourseInputBoundary.java

```diff
@@ -0,0 +1,13 @@
+package ports.usecases.account.RemoveSemesterCourse;
+
+public interface RemoveSemesterCourseInputBoundary {
+    /**
+     * Remove an existing course from the user's account's semester.
+     * @param username
+     * @param request
+     * @throws CourseDoesNotExistsError if the course requested does not exist in semester
+     */
+    void execute(String username, RemoveSemesterCourseRequest request) throws CourseDoesNotExistsError;
```

Comment:
If you're using a request model, it's a good idea to encapsulate the entire input into the request model. In this case, that means moving the `username` parameter to the request model.